doc/rados/operations/upmap.rst

```diff
@@ -12,21 +12,15 @@ clients understand the new *pg-upmap* structure in the OSDMap.
 Enabling
 --------
 
-To allow use of the feature, you must tell the cluster that it only
-needs to support luminous (and newer) clients with::
+This module is on by default. The cluster must only have luminous (and newer)
+clients. You can the turn the balancer off with::
 
-  ceph osd set-require-min-compat-client luminous
```

Comment:
this part is still relevant for upgraded clusters, I'd leave it - it's also the only documentation of this command